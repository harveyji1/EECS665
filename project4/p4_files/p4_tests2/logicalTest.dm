funct : () void {
	x : perfect int;
	y : int;
	give x;
	take y;
	isCold : bool = too hot;

    if(x and y){
    }
    if(x or y){

    }else{}
    
}