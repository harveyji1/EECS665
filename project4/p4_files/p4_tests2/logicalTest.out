funct{()->void} : () void  {
    x{perfectint} : perfect int;
    y{int} : int;
    give x{perfectint};
    take y{int};
    isCold{bool} : bool = false;
    if (x{perfectint} and y{int}){
    }
    if (x{perfectint} or y{int}){
    } else {
    }
}
