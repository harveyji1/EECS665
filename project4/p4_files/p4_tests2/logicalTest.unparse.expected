funct{()->void} : () void  {
    x{perfect int} : perfect int;
    y{int} : int;
    give x{perfect int};
    take y{int};
    isCold{bool} : bool = false;
    if (x{perfect int} and y{int}){
    }
    if (x{perfect int} or y{int}){
    } else {
    }
}
